ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-health-template.html

```diff
@@ -109,7 +111,7 @@
                                     <td md-cell>
                                         {{service.alertsCount}}
                                     </td>
-                                    <td md-cell >{{service.checkDate |  date : "MM/dd/yyyy 'at' h:mm"}}</td>
+                                    <td md-cell >{{'service-component-detail.time-locale' | translate:{date:service.checkDate} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:service.checkDate} }}                                    </td>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.